src/bridge/cockpitdbusjson.c

```diff
@@ -122,6 +122,30 @@ check_type (JsonNode *node,
   return TRUE;
 }
 
+static gboolean
+check_int_type (JsonNode *node,
+                const char *type_name,
```

Comment:
Oh right, I thought `get_type_string()` would give you something more readable, sorry:
```sh
$ python3 -c 'from gi.repository import GLib; print(GLib.Variant.new_int64(1).get_type_string())'
x
```
That's of course less than ideal. So I'm fine with keeping the manual strings, sorry for not checking this more closely before!